### ISSUE TITLE
Bug: ERROR getting assigned IP of eth0: IP address not found for interface: interface eth0 in 1 addresses

### ISSUE THREAD

**Issue by teejo75** [qdm12/gluetun#1588]
### Is this urgent?

No

### Host OS

Debian Bullseye

### CPU arch

x86_64

### VPN service provider

AirVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2023-05-22T06:07:02.257Z (commit eed62fd)

### What's the problem 🤔

Container won't start.
If I change tag to v3.34.0 then container starts.

### Share your logs

```plain text
gluetun  | ========================================
gluetun  | ========================================
gluetun  | =============== gluetun ================
gluetun  | ========================================
gluetun  | =========== Made with ❤️ by ============
gluetun  | ======= https://github.com/qdm12 =======
gluetun  | ========================================
gluetun  | ========================================
gluetun  | 
gluetun  | Running version latest built on 2023-05-22T06:07:02.257Z (commit eed62fd)
gluetun  | 
gluetun  | 🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
gluetun  | 🐛 Bug? https://github.com/qdm12/gluetun/issues/new
gluetun  | ✨ New feature? https://github.com/qdm12/gluetun/issues/new
gluetun  | ☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
gluetun  | 💻 Email? quentin.mcgaw@gmail.com
gluetun  | 💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
gluetun  | 2023-05-22T16:46:01+10:00 ERROR getting assigned IP of eth0: IP address not found for interface: interface eth0 in 1 addresses
gluetun  | 2023-05-22T16:46:01+10:00 INFO Shutdown successful
```

### Share your configuration

```yml
gluetun:
    image: qmcgaw/gluetun
    container_name: gluetun
    # line above must be uncommented to allow external containers to connect. See https://github.com/qdm12/gluetun/wiki/Connect-a-container-to-gluetun#external-container-to-gluetun
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - 8888:8888/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks
      - 8080:8080 # QBT Web UI
    volumes:
      - ./gluetun-config:/gluetun
    environment:
      # Timezone for accurate log times
      - TZ=xxx
      # See https://github.com/qdm12/gluetun/wiki
      - VPN_SERVICE_PROVIDER=airvpn
      - VPN_TYPE=wireguard
      - WIREGUARD_PRIVATE_KEY=xxx
      - WIREGUARD_PRESHARED_KEY=xxx
      - WIREGUARD_ADDRESSES=xxx
      - SERVER_REGIONS=xxx
    restart: unless-stopped
```

**Comment by Prozpe** [comment]
the same error for me...

**Comment by j4v3l** [comment]
I have the same issue:

---
version: "3"
services:
  gluetun:
    image: qmcgaw/gluetun
    container_name: gluetun
    # line above must be uncommented to allow external containers to connect. See https://github.com/qdm12/gluetun/wiki/Connect-a-container-to-gluetun#external-container-to-gluetun
    cap_add:
      - NET_ADMIN
    ports:
      - 8888:8888/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks
      - 6881:6881/udp
      - 6881:6881/tcp
      - 8080:8080/tcp
      - 9091:9091
      - 51413:51413
      - 51413:51413/udp
    volumes:
      - /root/.docker/configs/gluetun:/gluetun
    environment:
      # See https://github.com/qdm12/gluetun/wiki
      - VPN_SERVICE_PROVIDER=nordvpn
      - VPN_TYPE=openvpn
      # OpenVPN:
      - OPENVPN_USER=
      - OPENVPN_PASSWORD=
      - SERVER_REGIONS=United States
      - TZ=America/New_York

**Comment by seth100** [comment]
Same to me! It's referring to `eth0` but my network interface is not named that way!

**Comment by J3m5** [comment]
I got the same error.
I reverted to v3.33, issue is gone.

**Comment by qdm12** [comment]
Whoops sorry, latest commit broke latest, fixed in 3100cc1e5ee7d6523cc02e1c874a271713c13395

Note there is also v3.34.1 that should work fine.
